src/client/components/Message/Message.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Message.css';
+import PropTypes from 'prop-types';
+
+// message:{text, senderId,}, userId
+// Was the message sent by the current user. If so, add a css class
+// here message is object which contains user which need not be logged in and the text message
+const Message = ({ message, username }) => {
```

Comment:
```suggestion
const Message = ({ text, username }) => {
```

instead of message that is a object, just have the text that is a string